File: Final Code Python/Calculator/notepad_test.py
import unittest
from appium import webdriver
from notepad import notepad_PO
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.common.keys import Keys

class notepad_test(unittest.TestCase):
    notepad = None;

    def setUp(self):
        print("setup")
        desired_caps = {}
        desired_caps["app"] = "c:\\windows\\system32\\notepad.exe"
        self.notepad = webdriver.Remote(
            command_executor="http://127.0.0.1:4723",
            desired_capabilities=desired_caps
        )

    def tearDown(self):
        print("teardown")
        self.notepad.quit()

    def test_Notepad(self):
        print("notepad")
        np = notepad_PO.notepad_PO(self.notepad)
        self.notepad.find_element_by_name("Edit").send_keys("windriver is awesome")
        np.textArea().send_keys("windriver is awesome")
        np.textArea().send_keys(Keys.ALT, Keys.F4)
        np.dialogCancel().click()
        np.textArea().send_keys(Keys.ALT, Keys.F4)
        np.dialogDontSave().click()

File: Final Code Java/src/test/java/Notepad_test.java
import io.appium.java_client.windows.WindowsDriver;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.remote.DesiredCapabilities;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

import java.net.MalformedURLException;
import java.net.URL;

public class Notepad_test {
    private WindowsDriver notepad = null;
    private Notepad_PO np = null;
    @BeforeClass
    public void setup(){
        System.out.println("setup");
        DesiredCapabilities capabilities = new DesiredCapabilities();
        capabilities.setCapability("app","c:\\windows\\system32\\notepad.exe");
        try {
            notepad = new WindowsDriver(new URL("http://127.0.0.1:4723"), capabilities);
            np = new Notepad_PO(notepad);
        } catch (MalformedURLException e) {
            e.printStackTrace();
        }
    }

    @AfterClass
    public void teardown(){
        System.out.println("teardown");
        notepad.quit();
    }

    @Test
    public void testNotepad(){
        System.out.println("test");
        np.textArea().sendKeys("Windriver is awesome");
        Assert.assertEquals(np.textArea().getText(), "Windriver is awesome");
        np.textArea().sendKeys(Keys.ALT, Keys.F4);
        np.dialogCancel().click();
        np.textArea().sendKeys(Keys.ALT, Keys.F4);
        np.dialogDontSave().click();
    }
}
